maintainers/categories-set.nix

```diff
@@ -0,0 +1,766 @@
+/* List of categories
+    ```nix
+    handle = {
+      # Required
+      name = "Category Name";
+      description = ''
+        Description of category
+      '';
+      # Optional
+      relatedCategories = [];
+    };
+    ```
+
+    where
+
+    - `handle` is the handle you are going to use in nixpkgs expressions;
+       usually the same as the name but in camelCase
+    - `name` is the category name
+    - `description` is a description of the category
+    - `relatedCategories` is a list of related categories
```

Comment:
As names or as attribute set references?

Reply:
As attrset references.

Reply:
If there is no example for that, maybe mention this expectation.

Reply:
A practical example is not too hard. `audioVideo` is clearly related to `audio` and `video`.

Reply:
Done, I believe.